python/pysrc/kaskada/__init__.py

```diff
@@ -10,6 +10,8 @@
 from ._timestream import Literal
 from ._timestream import Timestream
 from ._timestream import record
+from .udf import udf
+from .udf import Udf
```

Comment:
Exposing the `Udf` class so we can test `isinstance(.., kd.Udf)`

Reply:
We shouldn't need to expose that externally -- the place where we do the `isinstance` is internal, so it can just import `Udf` from `udf`.